python_modules/automation/automation/docker/images/dagster-base/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG BASE_IMAGE
+FROM "${BASE_IMAGE}"
+
+ARG DAGSTER_VERSION
+
+# ==> Dagster base layer
+RUN \
+# Cron - remove along with dagster-cron once daemon scheduler is default
+       apt-get update -yqq \
+    && apt-get install -yqq --no-install-recommends cron \
+# Dagster
+    && pip install \
+        dagster==${DAGSTER_VERSION} \
+        dagster-graphql==${DAGSTER_VERSION} \
+        dagster-postgres==${DAGSTER_VERSION} \
+        dagster-aws==${DAGSTER_VERSION} \
```

Comment:
I'm torn about whether `dagster-{aws,gcp,azure}` should be included in the base layer; or whether we should have an intermediate "cloud-provider" layer used by all the k8s images.

Do y'all think using a Docker image and NOT wanting to use S3 is a common use case?

Reply:
It looks like every one of the other images use `dagster-aws`, but I don't know exactly why. If the use case of the image `dagster-base` is only within the scope of automation within the Dagster project and our CI/CD, then it'd be find to keep `pip install dagster-aws` in `dagster-base`.

On the other hand, if users of Dagster are going to build their images off of `dagster-base`, then we'll need to re-evaluate.

@johannkm Do you have any insight as to why there's `pip install dagster-aws` for every single Docker image in automation? Also, do you know if we intend to have users of Dagster build images on top of these images in `automation`?

Reply:
The provided presets use s3 storage.

> do you know if we intend to have users of Dagster build images on top of these images

That's an option, but I think most users either solely use these images or build their own (using these as reference) once they need something beyond what's provided here